vscode-trace-extension/src/trace-viewer-panel/trace-viewer-webview-panel.ts

```diff
@@ -162,22 +261,20 @@ export class TraceViewerPanel {
      * @param {unknown} data - payload
      */
     public static postMessageToWebviews(command: string, data: unknown): void {
+        // TODO replace by new messenger
```

Comment:
Maybe leftover? Or at least Messenger is used under-the-hood

Reply:
Correct. I removed it.